.github/workflows/ci_weekly_run.yaml

```diff
@@ -0,0 +1,21 @@
+name: weekly CI test run
+on:
+  workflow_call:
+    inputs:
+      run_number:
+        required: true
+        type: string
+
+concurrency:
+  group: ci-run-${{ inputs.run_number }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
```

Comment:
can we not automatically trigger the same stuff that "push" on master triggers? looks like this is missing the "docs" workflow, but would be nice to automatically detect this

Reply:
Took a quick look into whether this was possible. I didn't come across a clear straightforward solution. I was hoping we could just filter all workflows by what events could trigger them but they don't provide that information in the workflow endpoints: https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#list-repository-workflows

And I removed the docs workflow since we would only want to focus on `selfdrive` and `tools` since those are what gets updated the most and has a higher chance at producing flaky tests.